libs/blocks/merch/upgrade.js

```diff
@@ -46,21 +54,26 @@ export const handleIFrameEvents = ({ data: msgData }) => {
   if (app !== 'ManagePlan') return;
   switch (subType) {
     case MANAGE_PLAN_MSG_SUBTYPE.AppLoaded:
-      // todo hide spinner
+      document.querySelector('.upgrade-flow-content iframe')?.classList?.remove('loading');
+      document.querySelector('.upgrade-flow-content sp-theme')?.remove();
+      lanaLog('Showing modal', subType);
```

Comment:
Is there a business requirement to log all these operations with lana? Looks pretty verbose to me.

Reply:
@yesil do you know?

Reply:
This is an external application and we need to know when and where things break in order to be able to qualify an issue.
I see no harm, as this is only for logged-in users that click a button that is displayed only under certain conditions.